### DISCUSSION TITLE
Selectable Entries Condition depending on field value

### DISCUSSION THREAD

**Discussion by tonioseiler** [craftcms/cms#12044] [Ideas]
Hi,

I have a entries field and would like to make sure, that the list of selectable entries depends on a field values of the currenty edited entry. What can be the syntax of this.
<img width="822" alt="grafik" src="https://user-images.githubusercontent.com/4467443/193577631-a8558088-0c18-45c7-8dec-f3abefe4639e.png">

**Comment by brandonkelly** [comment]
Not sure I’m following. You only want to be able to select elements that are already related to the entry? How would you relate the first element on a new entry then?

**Reply by gvjacob** [reply]
This does seem counter-intuitive but I think I have a relevant example. An entry has a field for referencing multiple topics from a Topic section for example. There's another field to choose the "primary" topic out of that same list. While we can let editors select any topic in that single entry field, it does make for a better editor experience to narrow down the options to the topics the entry is already related to.

It doesn't seem like the Twig template for the `Related To` conditional has access to the currently edited entry. I've tried `{id}`, `{entry.id}` and `{{ object.id }}`.

**Reply by brandonkelly** [reply]
Aha, that makes sense.

Just tested `{id}` locally and it’s working as expected on my end.

**Reply by gvjacob** [reply]
Thanks for the reply!

You are right. It seems to work for a field that's directly attached to an entry. It doesn't seem to work for a field within a Content Block however. Those two `Praises` fields are the same fields. One of them is just nested inside a Content Block. Maybe this is considered expected? Not sure if internally a Content Block is considered an entry itself and is used as the entry context in the template.

https://github.com/user-attachments/assets/98077554-d880-4678-94cb-a2c66746e880

**Reply by brandonkelly** [reply]
Right, the content block is its own nested element within the entry. You can exclude elements related to the owner element using `{owner.id}`.
